api/client/proto/authservice.proto

```diff
@@ -1650,7 +1651,11 @@ service AuthService {
     rpc MaintainSessionPresence(stream PresenceMFAChallengeSend)
         returns (stream MFAAuthenticateChallenge);
 
+    // UpsertSessionTracker upserts a new session tracker resource.
+    rpc UpsertSessionTracker(types.SessionTrackerV1) returns (google.protobuf.Empty);
```

Comment:
We're actually going the other way around and deprecating Upserts in favor of Create+Update because it gives more control to the client. Any reason you're doing the opposite?

Reply:
Ah I thought that upsert over create/update was the latest decision, but last time I'd checked was probably half a year ago.

I really just wanted to change it to `CreateSessionTracker types.SessionTrackerV1) returns (google.protobuf.Empty)` instead of `CreateSessionTracker(CreateSessionTrackerRequest) returns (types.SessionTrackerV1)`, since CreateSessionTrackerRequest is just a copy of `types.SessionTrackerV1`, making it clunky to use.

What if instead I just added `types.SessionTrackerV1` to `CreateSessionTrackerRequest` and deprecate the old flatted out fields in `CreateSessionTrackerRequest`?